relnotes/resume.migration.md

```diff
@@ -0,0 +1,8 @@
+## Disallow direct resume in termination hooks
+
+It is now necessary to use `IScheduler.delayedResume` method if one task has to
+be resumed as a result of termination of another task. Attempting direct resume
+will result in a `SanityException` being thrown.
+
+All ocean facilities have been already adjusted to switch accordingly (for
+example, `await`).
```

Comment:
Suggest ", but please pay attention to adjusting application code that does the same."